Objects/dictobject.c

```diff
@@ -1737,7 +1736,41 @@ _PyDict_GetItem_KnownHash(PyObject *op, PyObject *key, Py_hash_t hash)
 
     ix = _Py_dict_lookup(mp, key, hash, &value);
     assert(ix >= 0 || value == NULL);
-    return value;
+    return value;  // borrowed reference
+}
+
+// Return the object from dictionary *op* which has a key *key*.
+// - If the key is present, set *pvalue to a new strong reference to the value
```

Comment:
`pvalue` or `result`?

Reply:
Oh, I missed that comment. I updated the comment and moved the comment to the header file to make it more visible (more people read header files, than people reading C files).